src/components/FormikControler/FormikControler.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import InputTextarea from './Input';
+import Textarea from "./Textarea";
+
+const FormikControler = (props) => {
+    const { control, ...rest } = props
+    switch (control) {
```

Comment:
I dont really like how you realise this component. In my opinion, its better to use `condition ? ... : ...`